pre-commit

```diff
@@ -19,6 +19,14 @@ if [ ! -e "$DEV_LIB_PATH/check-diff.sh" ]; then
 	exit 1
 fi
 
+DEV_LIB_COMMIT_MESSAGE_FILE=".dev-lib-commit-message.$PPID"
+DEV_LIB_COMMIT_MESSAGE=''
+function add_commit_message {
+	label=$1
+	value=$2
+	DEV_LIB_COMMIT_MESSAGE+="\n$label: $value"
```

Comment:
Temporary files will be garbage-collected with the next reboot, but what about users that use this tool in some server environment which isn't rebooted often?